acceptance-tests/journey/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM gradle:jdk11
 RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && unzip awscliv2.zip && ./aws/install
 COPY . .
-ENTRYPOINT ["./run-tests.sh"]
+RUN mv run-tests.sh /run-tests.sh
```

Comment:
Perhaps you could use the `WORKDIR` here in future then you'll be able to do `RUN ./run-test.sh`

Reply:
@jkunle That didn't work for me, as the java build stuff needs to be in /home/gradle and the script in root